src/polyfills/details/_base.scss

```diff
@@ -6,6 +6,8 @@
 summary {
 	// Make sure summary remains visible
 	display: list-item !important;
+	list-style-type: none; // Prevent list bullets from appearing in browsers that lack native details/summary support (e.g. IE/Edge)
+	list-style-type: disclosure-closed; // Show summary arrows in Firefox 49.0+ and future versions of Blink/Webkit
```

Comment:
I remark that in this version you removed the ```inside``` instruction that was after the ```disclosure-closed```. Should that fix the issue I saw in Edge where the bullet was show beside the arrow?